### PR TITLE
Added some triangle symbols

### PR DESCRIPTION
**Pull request by moulikagrawal**

### CODE REVIEW

**Comment by DenverCoder1** [comment]
Hi, there is already an issue open for this.

If you'd like to contribute, maybe try something new, for example these triangles

```
\bigblacktriangledown	▼	
\bigblacktriangleup	▲	
\blacktriangle	▴	
\blacktriangledown	▾	
\blacktriangleleft	◀	
\blacktriangleright	▶	
```

**Comment by moulikagrawal** [comment]
@DenverCoder1 sir kindly check now

**Comment by DenverCoder1** [comment]
This is nice, some small issues with the formatting though. Make sure that the only things separating the symbols are tabs. You can copy paste the tabs from other lines to make sure you get it right.

If you need help with it, let me know.

**Comment by moulikagrawal** [comment]
@DenverCoder1 kindly check now sir

File: dictionary.txt [hunk 1]
```diff
@@ -867,6 +867,12 @@ _j	ⱼ
 \llblacktriangle	◣	
 \ulblacktriangle	◤	
 \urblacktriangle	◥	
+\bigblacktriangledown	▼ 
+\bigblacktriangleup	▲	
+\blacktriangle		▴
+\blacktriangledown	▾ 
+\blacktriangleleft	◀ 
+\blacktriangleright	▶
```
Comment (DenverCoder1) [inline dictionary.txt:875]: The spaces are not correct

There should be 1 tab in between and 1 tab after as seen in the other examples

image


```suggestion
\bigblacktriangledown	▼	
\bigblacktriangleup	▲	
\blacktriangle	▴	
\blacktriangledown	▾	
\blacktriangleleft	◀	
\blacktriangleright	▶	
```

**Comment by moulikagrawal** [comment]
Sir i have changed as suggested

**Review by DenverCoder1 — APPROVED** [review]
Looks good! Thanks so much for contributing! :tada:

**Pull request merged**